src/community/geofence-server/src/main/java/org/geoserver/geofence/rest/UsersRestController.java

```diff
@@ -0,0 +1,151 @@
+package org.geoserver.geofence.rest;
```

Comment:
perhaps we should make a new module for spring rest, like restconfig-ng as we move things gradually. @jodygarnett  please advise